db/schema.rb

```diff
@@ -212,7 +220,8 @@
   end
 
   create_table "versions", force: :cascade do |t|
-    t.string "item_type", null: false
+    t.string "item_type"
+    t.string "{:null=>false}"
```

Comment:
Not sure what's going on here.  

`t.string "item_type", null: false`

Seems normal and right to me.  Never seen these get split like that before.